### PR TITLE
getLicense fixup 2

### PR DESCRIPTION
**Pull request by reinhardt**

### CODE REVIEW

**Review by ale-rt — COMMENTED** [review]
I am already on this as mentioned in the chat

**Comment by ale-rt** [comment]
Please @reinhardt check: https://github.com/syslabcom/recensio.plone/pull/73

File: src/recensio/plone/browser/review.py [hunk 1]
```diff
@@ -520,21 +519,14 @@ def getLicense(self):
...
+                    publication_licence = (
+                        licence_translated.text.output_relative_to(current) or ""
+                    )
                     licence_translated = licence_obj
                     publication_licence = licence_translated.text or ""
                     if publication_licence:
-                        publication_licence = publication_licence.output_relative_to(
-                            licence_translated
-                        )
+                        break
```
Comment (ale-rt) [inline src/recensio/plone/browser/review.py:527]: This change is not valid, we need to rendere the rich text value

**Review by ale-rt — CHANGES_REQUESTED** [review]

File: src/recensio/plone/browser/review.py (hunk 1 above)
Comment (reinhardt) [reply src/recensio/plone/browser/review.py:527]: That's done by line 524: `licence_translated.text.output_relative_to(current)`

Comment (ale-rt) [reply src/recensio/plone/browser/review.py:527]: But `text` can be `None` that's why I added it after another `if`

Comment (reinhardt) [reply src/recensio/plone/browser/review.py:525]: Like this? c81beb5

Comment (ale-rt) [reply src/recensio/plone/browser/review.py:525]: No, that's wrong because at that point it can already be a text... Leave it like it was before please

Comment (reinhardt) [reply src/recensio/plone/browser/review.py:525]: But you can't do `licence_translated.text or ""` and then call `output_relative_to` on that.

`AttributeError: 'str' object has no attribute 'output_relative_to'`

Also, you need to break when you've found something you want to use.

Comment (ale-rt) [reply src/recensio/plone/browser/review.py:525]: Nope because you call output_relative_to if text is something...

**Pull request closed without merging**